docker-compose.yml

```diff
@@ -57,6 +57,7 @@ services:
     volumes:
       - ./:/src:z
       - ./docker/message_broker/certs:/broker-certs:ro,z
+      - request-logs-volume:/var/log/iib-requests:z
```

Comment:
Should we keep this consistent with what we plan to have in the deployment?
`/var/log/iib/requests`